cuqi/samples/_samples.py

```diff
@@ -348,7 +362,7 @@ def hist_chain(self,variable_indices,*args,**kwargs):
         plt.legend(variables)
         return patches
 
-    def plot_ci(self,percent=95,exact=None,*args,plot_envelope_kwargs={},**kwargs):
+    def plot_ci(self, percent=95, exact=None, compute_on_par=True, *args, plot_envelope_kwargs={}, **kwargs):
```

Comment:
While we are at it, could you change plot_envelope_kwargs to None as default to avoid this issue with mutable defaults?

Reply:
Good point!